### ISSUE TITLE
get poles ui blocking script after closing

### ISSUE THREAD

**Issue by harzer99** [sdypy/sdypy-EMA#5]
Hi,
I tried plotting the stability chart for modal testing from a python file. The plotting works fine but the program won't continue executing after I closed tkinter window. Running the same code in a jupyter notebook works fine though.

```python
    # generates EMA.model from my data
    peaks = Peaks(tf_frf, pos_matrix)
    acc = peaks.acc
    acc.get_poles()
    acc.select_poles()

    # wont continue executing this part of the script
    print("finished")
    frf_rec, modal_const = acc.get_constants(whose_poles="own", FRF_ind="all")
    acc.print_modal_data()
```
my guess is that something is wrong with the code closes the tkinter window in the SelectPoles class. 

Appreciate your work!
Leonard

**Comment by jankoslavic** [comment]
@harzer99 thank you for this issue. Are you using Linux/Windows?

**Comment by klemengit** [comment]
I tested the code and indeed, there was an issue when executing scripts.
We have published a new release (v0.27.3).
It works on our end, can you please confirm that the new version works for you as well?

Thank you @harzer99 !

**Comment by harzer99** [comment]
Yes it works for me now! Thank you very much.
